arrocke/gloss-translation
# chore: Add Nx interactive executor

## What has changed

Updated nx-plugin executor


## Connected Issues


closes #254 
## QA Steps


1. run `nx prisma db migrate reset`
2. Ensure the following prompt is visible: `? Are you sure you want to reset your database? All data will be lost. » (y/N)`

## Post-Deployment



- [x ] Nothing required
